src/mlpack/tests/cf_test.cpp

```diff
@@ -548,5 +669,395 @@ BOOST_AUTO_TEST_CASE(SerializationTest)
   }
 }
 
+/**
+ * Make sure that correct number of recommendations are generated when query
+ * set for randomized SVD.
+ */
+BOOST_AUTO_TEST_CASE(CFGetRecommendationsAllUsersRandSVDTest)
+{
+  GetRecommendationsAllUsers<RandomizedSVDPolicy>();
```

Comment:
I guess, if we increase the ``iteratedPower`` and ``maxIterations`` and don't use the default values it might give us better results. In this case we would have to pass an instantization of the policy. Something like:

```c++
RandomizedSVDPolicy policy(10, 10);
```

might work.

Reply:
I was getting different error log when testing locally and thus wasn't able to debug. The jenkins log made more sense, and it was a silly enough mistake. :) Guess its kind of solved now. I'll push the changes and if this change is still necessary, do let me know.